pkg/utils/accelerators/tpu_test.go

```diff
@@ -0,0 +1,212 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package acceleratorutils
```

Comment:
we don't need to call the pkg this, there is util in the path already, lets just call it `accelerators`, can you also please change the file "tpu.go"?

Also, can you also please change the files under utils/statufulset as well?

Reply:
done